Sources/SwiftParser/Statements.swift

```diff
@@ -181,6 +186,44 @@ extension Parser {
 
     return RawConditionElementListSyntax(elements: elements, arena: self.arena)
   }
+  
+  /// Parse a list of condition elements returning optional trailing comma block.
+  mutating func _parseConditionListWithTrailingCommaBlock() -> (conditions: RawConditionElementListSyntax, blockAfterTrailingComma: RawClosureExprSyntax?) {
```

Comment:
I don’t think this is being used anywhere, is it?

Reply:
You're right,